### PR TITLE
Add links to the header

### PR DESCRIPTION
**Pull request by mjoyce91**
Add links to the header
- Logo = `/`
- Request Shields = `/`
- Print shields = `/work`
- Admin = `/requests`

Screen Shot 2020-04-03 at 11 18 05 PM

### CODE REVIEW

**Comment by mjoyce91** [comment]
@vadnguyen - I went ahead and knocked some of this out, but styling is not great

**Comment by vadnguyen** [comment]
@mjoyce91 styling rn, about to push out my updates

**Comment by vadnguyen** [comment]
@mjoyce91 - added some styling, updated to use NavLinks to leverage the activeClassName prop

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,20 +1,43 @@
...
+              </>
+            </Link>
           </Col>
 
+          {
+            user &&
+            <Col xs={6}>
+              <Row className="justify-content-between">
+                <Col className="my-auto font-weight-bold text-center">
+                  <NavLink to="/" activeClassName="active" className="nav-link">Request Shields</NavLink>
```
Comment (mjoyce91) [inline src/components/Navbar.tsx:30]: Looks like the active class for this one gets thrown off because all routes match `/`. However, we could easily change the home route to be `/home` or something

File: src/App.tsx [hunk 2]
```diff
@@ -50,6 +51,10 @@ const App: React.FC = () => {
                 <HomeView user={user} />
               </Route>
 
+              <Route path="/home">
```
Comment (ConnorDY) [inline src/App.tsx:54]: Why was this added?

Comment (mjoyce91) [reply src/App.tsx:54]: It fixes an issue where the bootstrap active class was always highlighting this URL.

If we go this route, I think we'll need an additional change to redirect the user to /home on login

Comment (vadnguyen) [reply src/App.tsx:55]: updated the login redirect to '/home' and the header logo too

**Comment by mjoyce91** [comment]
@ConnorDY - I updated this to match the changes you made with the role modal
